src/browser.ts

```diff
@@ -196,13 +241,45 @@ export class GrpcClient {
   }
 }
 
-export function lro(options) {
+/**
+ * Browser version of lro
+ *
+ * @param {Object=} options.auth - An instance of google-auth-library.
+ * When specified, this auth instance will be used instead of creating
+ * a new one.
+ * @param {Function=} options.promise - A constructor for a promise that
+ * implements the ES6 specification of promise. If not provided, native
+ * promises will be used.
+ * @return {Object} A OperationsClientBuilder that will return a OperationsClient
+ */
+export function lro(options: GrpcClientOptions) {
   options = Object.assign({scopes: []}, options);
   const gaxGrpc = new GrpcClient(options);
   return new OperationsClientBuilder(gaxGrpc);
 }
 
-// Wrapper function to throw exceptions on unsupported streaming calls
+/**
+ * Browser version of createApiCall
+ * A wrapper of createApiCall to throw exceptions on unsupported streaming calls
```

Comment:
This is true but this is not something that would be helpful for a person who sees this in its popup in VS Code :)

Let's move this line to the very end of this JSDoc comment and remove the wrapper wording. Let's just say `Throws exception on unsupported streaming calls`. For the user, the description of the regular `createApiCall` still applies and makes sense, and the added feature (throws exception) is just an added feature.